### ISSUE TITLE
Cannot input CJK in input bar(eg. search bar in Google page, login bar in facebook page)

### ISSUE THREAD

**Issue by lonicerae** [QupZilla/qupzilla#2198]
As subject. This issue doesn't affect the old version 1.6.6 provided by Debian. Any way for further troubleshoot on this? Thanks.

**Comment by lonicerae** [comment]
For example, if I type 'w' with the pinyin input method of ibus, candidate characters should appear, but with qupzilla 2.10, Chinese characters are not shown, only 'w' character was inputed in the input bar.

**Comment by mushroomsir** [comment]
https://bugreports.qt.io/browse/QTBUG-58362

**Comment by verycd88** [comment]
The same problem happens in Windows 7. Chinese (simplified) CAN be entered in the address bar of the browser and the search engine bar next to it. But in actual webpages, Chinese input DOES NOT work, for instance, search engines in amazon website, google search webpage, youtube website, etc.

**Comment by mushroomsir** [comment]
yes that's a known problem, please vote this [QTBUG-58362](https://bugreports.qt.io/browse/QTBUG-58362)

**Comment by TimothyXL** [comment]
Fixed upstream in QtWebEngine 5.9.
